flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
Can you build from source?

Reply:
Unfortunately I'm pretty unfamiliar with c# stuff and upstream [github action](https://github.com/Archeb/opentrace/actions/workflows/build-linux.yml) also has x86_64 configs only . And since this is an desktop program, seems like few users may use aarch64/riscv .